src/Automation/Interfaces/IScanner.cs

```diff
@@ -26,6 +27,23 @@ public interface IScanner
         /// <exception cref="AxeWindowsAutomationException"/>
         ScanResults Scan();
 
+        /// <summary>
+        /// Run AxeWindows automated tests on all windows.
+        /// </summary>
+        /// <remarks>
+        /// If a value was provided in <see cref="Config.OutputDirectory"/>,
+        /// output files may be written to the specified directory.
+        /// (Note: no output files will be written if no errors were found.)
```

Comment:
The comment here is correct for the single-root case, but the behavior is different for multi-root. That seems like it will just increase confusion. Would it be better for ScanAll to always writes the output files, even if it's a single-root case with no errors?

Reply:
That is a very good point. I have updated the behavior and now if EnableMultipleScanRoots option is enabled, we will write the output regardless if there are multiple outputs or not. One thought that just came to mind: What should happen if you don't enable multiple scan roots and still call ScanAll? Should that throw an exception like Scan does if EnableMultipleScanRoots are enabled? Or should the behavior in that case be that output files are written if there were issues with the first window?